### PR TITLE
Service privileges: API docs

### PR DESCRIPTION
**Pull request by thaJeztah**
This documents the Service privileges API changes, that were added in: 091b5e68ea735bf4e8ece708bbc8c413a32eab73

Carries the API changes of https://github.com/moby/moby/pull/32483, with some additional information about the values

Relates to https://github.com/moby/moby/pull/32339

### CODE REVIEW

**Comment by thaJeztah** [comment]
ping @aluzzardi @friism @johnstep PTAL

**Comment by thaJeztah** [comment]
Flakyness on WindowsRS1 https://jenkins.dockerproject.org/job/Docker-PRs-WoW-RS1/15503/console

```
02:17:12 ----------------------------------------------------------------------
02:17:12 FAIL: check_test.go:97: DockerSuite.TearDownTest
02:17:12 
02:17:12 check_test.go:98:
02:17:12     testEnv.Clean(c, dockerBinary)
02:17:12 environment/clean.go:67:
02:17:12     t.Fatalf("error removing containers %v : %v (%s)", containers, result.Error, result.Combined())
02:17:12 ... Error: error removing containers [b2e41ad5f83d] : exit status 1 (Error response from daemon: Could not kill running container b2e41ad5f83d1e270a9be7217544cd722d3a47bbf728d1b534dd7f334214d923, cannot remove - Cannot kill container b2e41ad5f83d1e270a9be7217544cd722d3a47bbf728d1b534dd7f334214d923: invalid container: b2e41ad5f83d1e270a9be7217544cd722d3a47bbf728d1b534dd7f334214d923
02:17:12 )
02:17:12 
02:17:12 
02:17:12 ----------------------------------------------------------------------
02:17:12 PANIC: docker_api_stats_test.go:62: DockerSuite.TestAPIStatsStoppedContainerInGoroutines
02:17:12 
02:17:12 ... Panic: Fixture has panicked (see related PANIC)
02:17:12 
```

**Review by boaz0 — APPROVED** [review]
👍 LGTM

File: api/swagger.yaml [hunk 1]
```diff
@@ -2015,6 +2015,56 @@ definitions:
...
+                  File:
+                    type: "string"
+                    description: |
+                      Load credential spec from this file. The file is read by the daemon, and must be present in the
+                      `CredentialSpecs` subdirectory in the docker data directory, which defaults to
+                      `C:\ProgramFiles\Docker\` on Windows.
+
+                      For example, specifying `spec.json` loads `C:\ProgramFiles\Docker\CredentialSpecs\spec.json`.
+
+                      <p><br /></p>
```
Comment (thaJeztah) [inline api/swagger.yaml]: Sorry, there are a bit dirty, but I couldn't find another way to add some spacing above the `note`

**Comment by thaJeztah** [comment]
ping @friism @johnstep @aluzzardi PTAL

**Comment by friism** [comment]
lgtm

**Review by johnstep — APPROVED** [review]
LGTM

**Comment by thaJeztah** [comment]
failing CI is unrelated; https://github.com/moby/moby/issues/34051

**Review by thaJeztah — COMMENTED** [review]
author LGTM

**Pull request merged**